spark/v3.1/spark/src/main/java/org/apache/iceberg/spark/procedures/ExpireSnapshotsProcedure.java

```diff
@@ -52,7 +52,8 @@ public class ExpireSnapshotsProcedure extends BaseProcedure {
         ProcedureParameter.optional("older_than", DataTypes.TimestampType),
         ProcedureParameter.optional("retain_last", DataTypes.IntegerType),
         ProcedureParameter.optional("max_concurrent_deletes", DataTypes.IntegerType),
-        ProcedureParameter.optional("stream_results", DataTypes.BooleanType)
+        ProcedureParameter.optional("stream_results", DataTypes.BooleanType),
+        ProcedureParameter.optional("snapshot_ids", DataTypes.createArrayType(DataTypes.LongType))
```

Comment:
This sets `containsNull` to true. I think that makes sense since you don't want to introduce casting problems, but it also means that `null` can be passed in and this will need to handle it rather than calling `toLongArray`.

Reply:
Looks like the parser can't actually handle nulls, so this is actually okay. I get a parser error when I try to test it.